public/global.css

```diff
@@ -107,3 +116,42 @@ a.button:focus {
 .button--large {
   padding: .625rem 3.75rem;
 }
+
+/**
+ * Utility classes for margin top
+ */
+.mt-s {
+  margin-top: 10px;
+}
+.mt-m {
+  margin-top: 20px;
+}
+.mt-l {
+  margin-top: 30px;
+}
+.mt-xl {
+  margin-top: 40px;
+}
+.mt-xxl {
+  margin-top: 50px;
+}
+
+/**
+ * Utility classes for margin bottom
+ */
+.mb-s {
+  margin-bottom: 10px;
+}
+.mb-m {
+  margin-bottom: 20px;
+}
+.mb-l {
+  margin-bottom: 30px;
+}
+.mb-xl {
+  margin-bottom: 40px;
+}
+.mb-xxl {
+  margin-bottom: 50px;
+}
```

Comment:
Honestly, I'd rather have all these values specified within the BEM-style classes, but having all of these sizes here makes it WAY easier to change — both to change the sizes, and to turn into CSS custom properties, if needed. Good job extracting it all here!